andela/AndelaEatsUI
# #166584926 Fix meals list

#### What does this PR do?
- Fixes the meals list.

#### Description of Task to be completed?
- The Meals list page has no pagination so is breaking because of a request to the wrong endpoint.

#### How should this be manually tested?
- Clone the repo and checkout the `bg-fix-meals-list-166584926` branch.
- Run the application and navigate to the `/admin/meals` endpoint.
- You should see the meal list.

#### What are the relevant pivotal tracker stories?
- [#166584926](https://www.pivotaltracker.com/story/show/166584926)
